src/drafts/CSSComponent/component.module.css

```diff
@@ -0,0 +1,6 @@
+.component {
+  background-color: tomato;
+  color: white;
+  width: fit-content;
+  padding: 0.625rem 0.75rem;
```

Comment:
not using primer/primitives yet, will create another follow up PR for that! (it has complexities related to theming)

Reply:
Happy to pair or collab async on this!